std/manual.md

```diff
@@ -667,7 +671,7 @@ Use `deno help` to see the help text.
 
 ```
 deno
-A secure runtime for JavaScript and TypeScript built with V8, Rust, and Tokio.
+A secure JavaScript and TypeScript runtime
```

Comment:
There is another one of these in the root README.

Reply:
I'll do that in another PR